core-web/libs/sdk/react/src/lib/components/DotcmsLayout/DotcmsLayout.spec.tsx

```diff
@@ -33,7 +33,12 @@ jest.mock('../PageProvider/PageProvider', () => {
 
 describe('DotcmsLayout', () => {
     it('renders correctly with PageProvider and rows', () => {
-        render(<DotcmsLayout pageContext={{ ...mockPageContext, isInsideEditor: true }} />);
+        render(
+            <DotcmsLayout
+                pageContext={{ ...mockPageContext, isInsideEditor: true }}
+                config={{ pathname: 'some-url' }}
```

Comment:
I am already padding the `pageContext` which contains the path, why do I need to pass it in the config?

Reply:
For the flow
pathname in config is for the framework route
The pageContext has the pageAsset and this is "obtained" using the pathname